onnx/defs/tensor/defs.cc

```diff
@@ -465,6 +465,16 @@ ONNX_OPERATOR_SET_SCHEMA(
             true,
             1,
             OpSchema::Differentiable)
+        .Input(
```

Comment:
Version bump is needed.

Reply:
@linkerzhang : The version for this op is already bumped to 13 as part of PR https://github.com/onnx/onnx/pull/2770